portal/src/texts/profile/colors.md

```diff
@@ -1,91 +1,93 @@
 
 - **Matt svart #3D3D3D** 
-Kan benyttes som en mørkere bakgrunnsfarge. Som bakgrunnsfarge tåler den hvit tekst.
+Vi kan bruke matt svart som en mørkere bakgrunnsfarge. Som bakgrunnsfarge kan den ha hvit tekst.
 
 - **Varm fjellgrå #696666**
-En mellomtone i gråskalaen. Benyttes på mindre flater og elementer, for å bryte opp layout. Fargen kan brukes med hvit tekst.
+Varm fjellgrå er en mellomtone i gråskalaen. Vi kan bruke den på  mindre flater og elementer, for å bryte opp layouten. På denne fargen kan vi bruke hvit tekst.
 
 - **Himmelgrå #F0F0F0**
-En lys grå, kan benyttes med svart tekst. Samme bruksområde som varm fjellgrå bare lysere. Kan brukes med svart tekst.
+Himmelgrå har samme bruksområde som varm fjellgrå, og vi bruker den med svart tekst.
 
 - **Snøhvit #FAFAFA**
-En hvitfarge. Brukes primært som bakgrunnsfarge eller på kort. Tåler matt svart eller mørkere for god kontrast.
+Snøhvit er en hvitfarge, som vi helst bare bruker som bakgrunnsfarge eller på kort. For å få god kontrast, fungerer den best med matt svart eller mørkere som kontrastfarge.
 
 
 ### Støttefarger
 
 - **Sand #DCB28C**
-En bruntone for bruk i illustrasjoner og grafikk .
+Sand er en bruntone, som vi kan bruke i illustrasjoner og grafikk.
 
 - **Jord #765D4B**
-En mørk bruntone for bruk i illustrasjoner og grafikk.
+Jord er en mørk bruntone, som passer til illustrasjoner og grafikk.
 
 - **Sol #FFC880**
-En gultone for bruk i illustrasjoner og grafikk. Sol er også en funksjonsfarge ment til å formidle nøytral informasjon. Tåler svart tekst. 
+Sol er fin til illustrasjoner og grafikk. Sol er også en funksjonsfarge, som vi kan bruke til å formidle nøytral informasjon. Den kan ha svart tekst. 
 
 - **Blomst #FF7059**
-En rødtone for bruk i illustrasjoner og grafikk. Blomst er også en funksjonsfarge der man trenger varselsfarge på mørke bakgrunner. 
+Blomst er en rødtone til bruk i illustrasjoner og grafikk. Blomst er også en funksjonsfarge, som er god som varselfarge på svart bakgrunn. 
 
 - **Skog #0D5745**
-En mørk grønn for bruk i illustrasjoner og grafikk. 
+Skog er en mørk grønn, til illustrasjoner og grafikk. 
 
 - **Hav #0A3166** 
-En mørk blå for bruk i illustrasjoner og grafikk.
+Hav er en mørk blåfarge, til illustrasjoner og grafikk.
 
 
 ### Funksjonsfarger
 
 - **Gress #50E68C**
-Funksjonsfarge for å formidle suksess. Benyttes kun mot svart eller matt svart.
+Gress er den funksjonsfargen vi bruker til å vise at noe er vellykket, for eksempel at brukeren er ferdig med en handling. Den bruker vi bare mot svart eller matt svart.
 
 - **Himmel #509CE6**
-Funksjonsfarge for å formidle informasjon fra systemet. Benyttes kun mot svart.
+Himmel er en av funksjonsfarge vi bruker til informasjonsmeldinger. Den brukes bare mot svart.
 
```

Comment:
Gress og Himmel er lagt som rene funksjonsfarger her men disse er også støttefarger som kan brukes i illustrasjoner.
Foreslår at vi flytter beskrivelsene av disse opp til støttefarger der de hører hjemme og beskriver funksjonsfargene omtrent slik. 
(Dette kan sannsynligvis skrives mye bedre men er en rask input til Gørild slik at hun forstår sammenhengen)

Under overskriften "Funksjonsfarger" kan vi beskrive fargene omtrent slik:

Funksjonsfarger for hvit bakgrunn: (Disse fargene er ikke tillatt å bruke i Illustrasjoner men er utviklet for å tilfredstille WCAG-krav))
- UX Blå
- UX Rød
- UX Grønn (for vi har vel fortsatt den grønne som ble definert?)

Funksjonsfarger for sort bakgrunn (dark mode):
Her bruker vi flg støttefarger:
- Blomst
- Himmel
- Gress

Reply:
Himmel og Gress var alltid presentert som funksjonsfarger, og har aldri vært presentert som støttefarger. Dette støattes også av tidligere arbeid jeg kan finne i Figma